<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #808080</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="lineEdit_lastname_student">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>160</y>
      <width>551</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffffff</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_Lesson_Edit_edit_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>270</y>
      <width>181</width>
      <height>28</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff</string>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_last_name_teacher">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>160</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff</string>
    </property>
    <property name="text">
     <string>Last Name:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_student_number">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>210</y>
      <width>551</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_first_name_student">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>110</y>
      <width>551</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:#ffffff
</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_first_name_teacher">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff</string>
    </property>
    <property name="text">
     <string>First Name:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username_teacher">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>210</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff</string>
    </property>
    <property name="text">
     <string>Student Number:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_username_teacher_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>380</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff</string>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_password_student">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>380</y>
      <width>551</width>
      <height>22</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: #ffffff</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
